### PR TITLE
Baseline cleanup and reorg based on Icosa Open Blocks fork

### PR DESCRIPTION
**Pull request by andybak**

### CODE REVIEW

File: Assets/ThirdParty/SteamVR/Editor/SteamVR_Editor.cs
Comment (ianmacgillivray) [inline Assets/ThirdParty/SteamVR/Editor/SteamVR_Editor.cs:1]: Can't add this, we don't have the rights to open source it

File: open-blocks.sln.DotSettings.user
Comment (ianmacgillivray) [inline open-blocks.sln.DotSettings.user:1]: Is this open-blocks specific? Commit says so.

**Review by ianmacgillivray — CHANGES_REQUESTED** [review]

File: Assets/ThirdParty/SteamVR/Editor/SteamVR_Editor.cs
Comment (andybak) [reply Assets/ThirdParty/SteamVR/Editor/SteamVR_Editor.cs:1]: As discussed on chat the same files were included in the Tilt Brush repo and the licence allows redistribution - so this is most likely ok.

File: open-blocks.sln.DotSettings.user
Comment (andybak) [reply open-blocks.sln.DotSettings.user:1]: Removed

**Review by ianmacgillivray — APPROVED** [review]

**Pull request merged**